app/views/_controls.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="masthead_controls" class="container-fluid">
+  <div class="row">
+    <div class="col-xs-12 col-sm-5 col-md-6">
+      <nav class="navbar navbar-default" role="navigation">
```

Comment:
It seems like this element should contain the top menu items (about, help, contact) while the elements that you have added (dashboard, files, browse view) belong to the ".breadcrumb" ul. 

Please refer to test site.